### DISCUSSION TITLE
App Crushing

### DISCUSSION THREAD

**Discussion by byaruhaf** [dataFude-app/dataFude#5] [Beta feedback] [answered]
The dataTile App shows some activity and then crashes immediately when running it with my app.
When running the sample app the dataTile App works as expected. 

https://user-images.githubusercontent.com/4116539/204518561-68b20532-32a5-4cf4-8e18-c585451ac4be.mov

The dataTile App only crushes when the option below is selected. 

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/4116539/204525778-8c5a08e9-5e11-49bf-bf03-893368f63b56.png">

**Comment by icanzilb** [comment]
Congrats, this is the first reported crasher 😁

I got the crash report and I'm going to track down where the issue is, thank you so much. Something in the log looks like a user logged value but probably contains some characters that break the regex that parses everything. I'll chime in here when I know more

**Comment by icanzilb** [comment]
@byaruhaf are you able to share the logs showing up in Xcode's console when you start the app? I have a code fix that'll prevent the crash but I'd love to find which log line was the one causing it so I can keep it mind also for the future.

**Comment by byaruhaf** [comment]
@icanzilb there is nothing printed out to the Xcode's console, but I can give you access to the app so you can hunt down the log line.

**Comment by icanzilb** [comment]
@byaruhaf This is going to be amazing. If I can reproduce I can fix this properly 👍🏼

**Comment by byaruhaf** [comment]
sent you an invite

**Comment by icanzilb** [comment, marked as answer]
Thanks @byaruhaf, I owe you one! Reproduced and caught the log line: 
<img width="663" alt="image" src="https://user-images.githubusercontent.com/633862/204557125-7a71cc8a-f323-4040-a151-da50894a98ae.png">

**Comment by icanzilb** [comment]
@byaruhaf Could you please update from TestFlight to the latest version 0.94(6) and check if that fixes the issue for you?

**Comment by byaruhaf** [comment]
Thank @icanzilb issue is fixed with the latest update
